src/pybkgmodel/processing.py

```diff
@@ -908,3 +909,40 @@ def bkg_map_maker(self, maker):
         if not isinstance(maker, ExclusionMap):
             raise TypeError(f"Maker must be of type {ExclusionMap}")
         BkgMakerBase.bkg_map_maker.fset(maker)
+
+class RunwiseOffDataMap(_Runwise):
+    def __init__(self, off_runs, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.off_runs = off_runs
+        self.excl_region    = [Regions.parse(reg,format='ds9') for reg in 
```

Comment:
Suggest removing the line break after `in`.

Reply:
I can introduce the line break earlier, but lines exceeding 100 characters are in disagreement with the python coding guidelines. Personally, I do not find this 100 character limit too important, so let me know what you think.